openfisca_us/tests/policy/baseline/states/tax/income/state_taxable_income.yaml

```diff
@@ -0,0 +1,22 @@
+- name: Default is zero.
+  period: 2022
+  output:
+    state_taxable_income: 0
+
+- name: Otherwise max(AGI - deductions - exemptions, 0)
+  period: 2022
+  input:
+    adjusted_gross_income: 10
+    state_income_tax_deductions: 2
+    state_income_tax_exemptions: 3
+  output:
+    state_taxable_income: 5
```

Comment:
Totally unnecessary but for tests which are formulas it might be neat to write:
`state_taxable_income: max(10 - 2 - 3, 0)`

Reply:
Done

Reply:
Actually I got an error with this.

I'd favor entering values, and then using comments to go through the math.